x-pack/plugins/task_manager/task_store.ts

```diff
@@ -347,16 +422,21 @@ function rawSource(doc: TaskInstance) {
     id,
     type: 'task',
     task: source,
+    kibana: {
+      uuid: store.getKibanaUuid(), // needs to be pulled live
+      version: TEMPLATE_VERSION,
+      apiVersion: API_VERSION,
+    },
   };
 }
 
-function taskDocToRaw(doc: ConcreteTaskInstance, index: string): RawTaskDoc {
-  const { type, task } = rawSource(doc);
+function taskDocToRaw(doc: ConcreteTaskInstance, store: TaskStore): RawTaskDoc {
+  const { type, task, kibana } = rawSource(doc, store);
 
   return {
     _id: doc.id,
     _index: index,
```

Comment:
yeah that was a bad merge resolution: 36dd2123d4

The `taskDocToRaw` thing is pre-existing code, if first-class functions are a problem then it would be +1 from me to do a cleanup PR. That's not for 6.7 though.

Reply:
Yeah, I think a cleanup PR would make sense, not sure the code is very clear on the usage of `taskDocToRaw` and `rawSource`. But not essential for 6.7